hawkbit-runtime/docker/generator/generate.sh

```diff
@@ -11,17 +11,20 @@ set -euxo pipefail
 
 HAWKBIT_VERSION="0.3.0M6"
 MARIADB_DRIVER_VERSION="2.5.3"
+POSTGRESQL_DRIVER_VERSION="42.2.10"
```

Comment:
Since there is no need anymore to add the driver within `Dockerfile` this variable and replacement can be removed as well